index.html

```diff
@@ -21,7 +21,7 @@ <h1 class="home__cover__heading">Savas Labs</h1><hr>
       <!-- Most recent case studies. -->
       <div class="case-studies">
         <h2>Recent work</h2>
-        {% for item in site.case-studies limit 3 %}
+        {% for item in site.case-studies limit:3 %}
```

Comment:
@chrisarusso Yes, the colon is required. I had the syntax wrong before.

We're going to have to update this once the Pauli Murray case study is up to choose the order; this is a temporary measure.